packages/codegen.go/src/example.ts

```diff
@@ -427,7 +429,6 @@ function generateFakeExample(goType: go.PossibleType, name?: string): go.Example
       case 'bool':
         return new go.BooleanExample(false, goType);
       case 'byte':
-      case 'string':
       case 'rune':
         return new go.StringExample(`<${name ?? 'test'}>`, goType);
```

Comment:
Should this still return a stringexample?

Reply:
Yeah `byte` and `rune` are still strings.